lib/mumuki/domain/submission/base.rb

```diff
@@ -52,8 +52,6 @@ def id
   private
 
   def save_submission!(assignment)
-    assignment.content = content
-    assignment.recontextualize!
     assignment.save!
```

Comment:
Does it make sense for Base to keep saving the assignment or not? I understand that it should only be in PersistentSubmission. I am right? @julian-berbel  @flbulgarelli